tests/util/mod.rs

```diff
@@ -172,8 +172,8 @@ pub fn assert_error<T, E: fmt::Display>(result: Result<T, E>, expected_msg: &str
         Err(err) => assert!(
             err.to_string().contains(expected_msg),
             "wanted: {}, got: {}",
+            expected_msg,
```

Comment:
@Thomasdezeeuw what am I missing? This is how a triggered assert looks like with the fix. 
```

---- unconnected_udp_socket_connected_methods stdout ----
thread 'unconnected_udp_socket_connected_methods' panicked at 'wanted: blablabla, got: A request to send or receive data was disallowed because the socket is not connected and (when sending on a datagram socket using a sendto call) no address was supplied. (os error 10057)', tests\util\mod.rs:172:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

```

Reply:
The fix seems correct to me, wanted is what you gave as argument to assert, right?

Reply:
Sorry you're right. Weird that nobody noticed.